feature-libs/product/configurators/common/components/group-menu/configurator-group-menu.component.html

```diff
@@ -23,10 +23,16 @@
             }}"
             [class.cx-menu-conflict]="isConflictGroupType(group.groupType)"
             [class.WARNING]="isConflictGroupType(group.groupType)"
+            [class.DISABLE]="
+              isGroupDisabled(configuration.owner, group.id) | async
```

Comment:
Why do we need this? We are looping the groups and could just query the configurable attribute, or am I missing something?